### PR TITLE
Add tbot recipe

### PR DESCRIPTION
**Pull request by saibhargav1508**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by saibhargav1508** [comment]
I am willing to be listed as the maintainer of this recipe

**Comment by saibhargav1508** [comment]
Not able to figure out what is going wrong with the windows build
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=212540&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=23010c01-6a9d-5f94-9037-792618644b77&l=81

File: recipes/tbot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "tbot" %}
+{% set version = "0.8.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Rahix/tbot.git
```
Comment (scopatz) [inline recipes/tbot/meta.yaml:10]: Please build from a source tarball

File: recipes/tbot/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{% set name = "tbot" %}
+{% set version = "0.8.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Rahix/tbot/archive/v0.8.2.tar.gz
```
Comment (scopatz) [inline recipes/tbot/meta.yaml:10]: This should be templated in order to receive  automatic updates: `https://github.com/Rahix/tbot/archive/v{{ version }}.tar.gz`

**Comment by saibhargav1508** [comment]
[noarch_python] Noarch package contains binary script: tbot.exe
@scopatz Does this mean I restrict the recipe only to Windows?

File: recipes/tbot/meta.yaml (hunk 2 above)
Comment (saibhargav1508) [reply recipes/tbot/meta.yaml]: Oh Yes. Sorry I missed it

**Comment by scopatz** [comment]
Noarch recipes only need to build on Linux. This is fine

**Pull request merged**